java/ci/build-in-docker.sh

```diff
@@ -86,6 +86,11 @@ if [ "$SIGN_FILE" == true ]; then
     BUILD_ARG="$BUILD_ARG -Prelease"
 fi
 
+if [ -f $WORKSPACE/java/ci/settings.xml ]; then
+    # Build with an internal settings.xml
+    BUILD_ARG="$BUILD_ARG -s ci/settings.xml"
```

Comment:
It's a bit weird to check for existence of a path then use a different path inside the conditional block.  Either this should use `$WORKSPACE/java/ci/settings.xml` or the check should use `ci/settings.xml`.

Reply:
Updated to use absolute path.